lib/matplotlib/backends/backend_cairo.py

```diff
@@ -204,9 +203,7 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
             ctx.select_font_face(*_cairo_font_args_from_font_prop(prop))
             ctx.set_font_size(self.points_to_pixels(prop.get_size_in_points()))
             opts = cairo.FontOptions()
-            opts.set_antialias(
-                cairo.ANTIALIAS_DEFAULT if mpl.rcParams["text.antialiased"]
-                else cairo.ANTIALIAS_NONE)
+            opts.set_antialias(gc.get_antialiased())
```

Comment:
Previously were passing in (what look like) enums from the cairo module, now we are passing a bool.

Would it be better to still have the ternary but just switched on a different value?

Reply:
`gc` is a `class GraphicsContextCairo(GraphicsContextBase)` object, which wraps over a `Cairo.Context` object.
When `gc.set_antialiased()` is called, the `Cairo.Context` object set the antialias state (managed by pycairo not us) to a Cairo enum. 
So in Cairo backend,  `gc.get_antialiased()` returns a Cairo enum from the `Cairo.Context` object, not a bool. So I guess we are good for the current implementation?

Also I found a mistake I made when looking at how state is set, which could be fixed by overriding `gc.get_antialiased()` in `class GraphicsContextCairo`.

Reply:
👍🏻 

I missed that this was `gc.get_antialiased` not `Text.get_antialiased`, but I am glad my confusion pointed you at a different problem.